### DISCUSSION TITLE
Log to new file every day

### DISCUSSION THREAD

**Discussion by UseRadio** [savonet/liquidsoap#1778] [Q&A]
Hello,

Is it possible to make LiquidSoap to log to a new file every day? Right now it's logging to same file all the time.
I could not find anything about it in the documentation.
Thanks in advance

**Comment by nigeyuk** [comment]
Would logrotate be able to do this?

**Comment by UseRadio** [comment]
@nigeyuk thanks for your input. 
Do you have any code snippet so I can test logrotate?

**Comment by nigeyuk** [comment]
Try something like this, and adjust for your log files location

/var/log/liquidsoap/liquidsoap.log {
    monthly
    create 0644 root root
    rotate 5
    size=1M
    dateext
    dateformat -%d%m%Y
    notifempty
    mail me@mydomain.com
}

Something like that ?   more info here 

https://www.tecmint.com/install-logrotate-to-manage-log-rotation-in-linux/
